### DISCUSSION TITLE
Help before I start making a SquishBox, please?

### DISCUSSION THREAD

**Discussion by viragored** [GeekFunkLabs/fluidpatcher#85] [Q&A] [answered]
_[I posted this question on [Youtube](https://www.youtube.com/watch?v=H0lJY4uJ5W4), before finding this Github. I hope this is the right place to post my questions]_
Hi - this looks great. I'm just putting together a shopping list to attempt a scratch-build. The video and kit show a "rotary + momentary push switch encoder", but the scratch-build shows only two button switches (no encoder). Is there a way to incorporate the rotary encoder with the perfboard build?
I know little about how electronics work so this might be a dumb question - should the trimpot be 1K or 10K, or doesn't it matter much? Your wiring diagram https://cdn.hackaday.io/images/5555181692640672364.jpg shows 1K; the components list https://hackaday.io/project/9097/components shows 10K.
Thanks - any help will be most welcome.

**Comment by albedozero** [comment, marked as answer]
I may have used a 1K pot in that very early build, but 10K is what's generally called for and what I've been using in my production builds. I think it doesn't matter too much, since it's just acting as a voltage divider. Maybe 10K is safer for other components because it reduces the current, but I would guess the difference is small.
